docs/packages/pkg/CLI11.rst

```diff
@@ -0,0 +1,26 @@
+.. spelling::
+
+    CLI11
+
+.. index:: commandline tools ; CLI11
+
+.. _pkg.CLI11:
+
+CLI11
+=====
+
+.. warning::
```

Comment:
Please use this template:
* https://raw.githubusercontent.com/ruslo/hunter/master/docs/packages/pkg/foo.rst

See:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation